designer/src/main/java/org/restcomm/connect/rvd/upgrade/UpgradeService.java

```diff
@@ -52,7 +51,7 @@
     }
 
     // valid project versions. If a version is not here it can either considered 'future' version or garbage.
-    static final String[] versionPath = new String[] {"rvd714","1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7", "1.8","1.9", "1.10", "1.11"};
+    static final String[] versionPath = new String[] {"rvd714","1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7", "1.8","1.9", "1.10", "1.11", "1.12"};
     // project versions where the project state .json file should be upgraded
     static final List<String> upgradesPath = Arrays.asList(new String [] {"1.0","1.6"});
```

Comment:
`upgradesPath` should include "1.12" if 1.11 are not forward compatible. This will happen if the Gather menu 'digits' has been renamed to 'key' in 1.12.